sdk/textanalytics/Azure.AI.TextAnalytics/src/ServiceClients/LanguageServiceClient.cs

```diff
@@ -16,87 +20,2189 @@ namespace Azure.AI.TextAnalytics.ServiceClients
+
+            var pipeline = HttpPipelineBuilder.Build(options, new BearerTokenAuthenticationPolicy(credential, authorizationScope));
+            _languageRestClient = new MicrosoftCognitiveLanguageServiceRestClient(_clientDiagnostics, pipeline, endpoint.AbsoluteUri, serviceVersion);
+        }
+
+        public LanguageServiceClient(Uri endpoint, AzureKeyCredential credential, string defaultLanguage, string defaultCountryHint, string authorizationScope, string serviceVersion, ClientOptions options)
+        {
+            Argument.AssertNotNull(endpoint, nameof(endpoint));
+            Argument.AssertNotNull(credential, nameof(credential));
+            Argument.AssertNotNull(options, nameof(options));
+
+            Argument.AssertNotNullOrEmpty(authorizationScope, nameof(authorizationScope));
+            Argument.AssertNotNullOrEmpty(serviceVersion, nameof(serviceVersion));
+
+            _baseUri = endpoint;
+            _clientDiagnostics = new TextAnalyticsClientDiagnostics(options);
+            _defaultLanguage = defaultLanguage;
+            _defaultCountryHint = defaultCountryHint;
+
+            var pipeline = HttpPipelineBuilder.Build(options, new AzureKeyCredentialPolicy(credential, Constants.AuthorizationHeader));
+            _languageRestClient = new MicrosoftCognitiveLanguageServiceRestClient(_clientDiagnostics, pipeline, endpoint.AbsoluteUri, serviceVersion);
+        }
+
         #region Detect Language
 
-        public override Task<Response<DetectedLanguage>> DetectLanguageAsync(string document, string countryHint = default, CancellationToken cancellationToken = default) => throw new NotImplementedException();
-        public override Response<DetectedLanguage> DetectLanguage(string document, string countryHint = default, CancellationToken cancellationToken = default) => throw new NotImplementedException();
-        public override Task<Response<DetectLanguageResultCollection>> DetectLanguageBatchAsync(IEnumerable<string> documents, string countryHint = default, TextAnalyticsRequestOptions options = default, CancellationToken cancellationToken = default) => throw new NotImplementedException();
-        public override Response<DetectLanguageResultCollection> DetectLanguageBatch(IEnumerable<string> documents, string countryHint = default, TextAnalyticsRequestOptions options = default, CancellationToken cancellationToken = default) => throw new NotImplementedException();
-        public override Task<Response<DetectLanguageResultCollection>> DetectLanguageBatchAsync(IEnumerable<DetectLanguageInput> documents, TextAnalyticsRequestOptions options = default, CancellationToken cancellationToken = default) => throw new NotImplementedException();
-        public override Response<DetectLanguageResultCollection> DetectLanguageBatch(IEnumerable<DetectLanguageInput> documents, TextAnalyticsRequestOptions options = default, CancellationToken cancellationToken = default) => throw new NotImplementedException();
+        /// <summary>
```

Comment:
Currently `TextAnalyticsClient` and `LanguageServiceClient` have them. I think Intellisense will use the override, so will it be better to have the docstrings moved from `TextAnalyticsClient` to the `LegacyServiceClient`?